sftp-connector/pom.xml

```diff
@@ -5,12 +5,14 @@
   <artifactId>sftp-connector</artifactId>
   <version>9.2.3-SNAPSHOT</version>
   <packaging>iar</packaging>
+  <properties>
+    <spring.version>5.2.5.RELEASE</spring.version>
+  </properties>
   <dependencies>
-    <!-- https://mvnrepository.com/artifact/com.jcraft/jsch -->
     <dependency>
-      <groupId>com.jcraft</groupId>
-      <artifactId>jsch</artifactId>
-      <version>0.1.55</version>
+      <groupId>org.springframework.integration</groupId>
```

Comment:
I just saw that this seemingly small dependency brings a whole bunch of sprint libs with it. It would be cool to reduce the amount of required libraries. You can do so by opening the MavenPomEditor (right click -> openWidth) and right clicking on dependencies that may are not used at runtime.

At least 'commons-io' should not be required. As it is already present in the axonivy-core.
![sftpDeps](https://user-images.githubusercontent.com/15085693/129369472-e227d4d5-125d-44a3-b45a-c41d08ad93ab.png)

Reply:
You may have to experiment a little: often many of the supporting libraries can be ommitted as you do not run in an spring context. Just exclude some that seem not to be required (e.g. spring-aop) and test whether the SFTP client still runs.

Reply:
In this case, I will need to dive deep into the spring source code to isolate the real used jars from the not used ones and it will be risky for Connector. But I will try and see if I can do it.

Reply:
Don't investigate too much on this. I'd just test whether our demonstrated use-case also works with less library deps. Things like spring-messaging, spring-jcl, spring-tx sound totally out of context.